leapp/exceptions.py

```diff
@@ -109,7 +109,9 @@ def __init__(self, command):
 
 
 class LeappRuntimeError(LeappError):
-    pass
+    def __init__(self, message, exception_info=None):
+        super().__init__(message)
```

Comment:
This is not python2.7 compatible, which is required, that's why the test is failing
```suggestion
        super(LeappRuntimeError, self).__init__(message)
```